.github/workflows/tests.yml

```
@@ -322,7 +322,7 @@ jobs:
         working-directory: complement/dockerfiles
 
       # Run Complement
-      - run: go test -v -tags synapse_blacklist ./tests
+      - run: go test -v -tags synapse_blacklist,msc2946,msc3083,msc2403 ./tests
```

For easier scanning of the list:

```suggestion
      - run: go test -v -tags synapse_blacklist,msc2403,msc2946,msc3083 ./tests
```